website/src/release.md

```diff
@@ -0,0 +1,383 @@
+
+To learn more about apache iceberg, please see https://rust.iceberg.apache.org/
+
+Checklist for reference:
+
+[ ] Download links are valid.
+[ ] Checksums and signatures.
+[ ] LICENSE/NOTICE files exist
+[ ] No unexpected binary files
+[ ] All source files have ASF headers
+[ ] Can compile from source
+
+More detailed checklist please refer to:
+https://github.com/apache/iceberg-rust/tree/main/scripts
+
+To compile from source, please refer to:
+https://github.com/apache/iceberg-rust/blob/main/CONTRIBUTING.md
+
+Here is a Python script in release to help you verify the release candidate:
+
+./scripts/verify.py
+
+Thanks
+
+${name}
+```
+
+Example: <https://lists.apache.org/thread/c211gqq2yl15jbxqk4rcnq1bdqltjm5l>
+
+After at least 3 `+1` binding vote (from Iceberg PMC member), claim the vote result:
+
+Title:
+
+```
+[RESULT][VOTE] Release Apache Iceberg Rust ${release_version} - Round 1
+```
+
+Content:
+
+```
+Hello, Apache Iceberg Rust Community,
+
+The vote to release Apache Iceberg Rust ${release_version} has passed.
+
+The vote PASSED with 3 +1 binding and 1 +1 non-binding votes, no +0 or -1 votes:
+
+Binding votes:
+
+- xxx
+- yyy
+- zzz
+
+Non-Binding votes:
+
+- aaa
+
+Vote thread: ${vote_thread_url}
+
+Thanks
+
+${name}
+```
+
+Example: <https://lists.apache.org/thread/xk5myl10mztcfotn59oo59s4ckvojds6>
+
+## Official Release
+
+### Push the release git tag
+
+```shell
+# Checkout the tags that passed VOTE
+git checkout ${release_version}
+# Tag with the iceberg version
+git tag -s ${iceberg_version}
+# Push tags to github to trigger releases
+git push origin ${iceberg_version}
+```
+
+### Publish artifacts to SVN RELEASE branch
+
+```shell
+svn mv https://dist.apache.org/repos/dist/dev/iceberg/iceberg-rust/${release_version} https://dist.apache.org/repos/dist/release/iceberg/iceberg-rust/${iceberg_version} -m "Release ${iceberg_version}"
+```
+
+### Change Iceberg Rust Website download link
+
+Update the download link in `website/src/download.md` to the new release version.
+
+### Create a GitHub Release
+
+- Click [here](https://github.com/apache/iceberg-rust/releases/new) to create a new release.
+- Pick the git tag of this release version from the dropdown menu.
+- Make sure the branch target is `main`.
+- Generate the release note by clicking the `Generate release notes` button.
+- Add the release note from every component's `upgrade.md` if there are breaking changes before the content generated by GitHub. Check them carefully.
+- Publish the release.
+
+### Send the announcement
+
+Send the release announcement to `dev@iceberg.apache.org` and CC `announce@apache.org`.
+
+Instead of adding breaking changes, let's include the new features as "notable changes" in the announcement.
+
+Title:
+
+```
+[ANNOUNCE] Release Apache Iceberg Rust ${iceberg_version}
+```
+
+Content:
+
+```
+Hi all,
+
+The Apache Iceberg rust community is pleased to announce
+that Apache Iceberg rust ${iceberg_version} has been released!
```

Comment:
I'm a bit of a nitpicker on this one, but I noticed that we don't have a consistent way of capitalizing Apache, Iceberg and Rust.
```suggestion
The Apache Iceberg Rust community is pleased to announce
that Apache Iceberg Rust ${iceberg_version} has been released!
```

Reply:
I'm a little confusing about the phrase `Apache Iceberg Rust community`, maybe we should just say `Apache Iceberg community`?

Reply:
I'm fine with both of them.